radio/src/gui/common/stdlcd/radio_sdmanager.cpp

```diff
@@ -175,12 +175,32 @@ void menuRadioSdManager(event_t _event)
     audioQueue.stopSD();
 #endif
     BYTE work[_MAX_SS];
-    if (f_mkfs(0, FM_FAT32, 0, work, sizeof(work)) == FR_OK) {
-      f_chdir("/");
-      REFRESH_FILES();
-    }
-    else {
-      POPUP_WARNING(STR_SDCARD_ERROR);
+    FRESULT res = f_mkfs("", FM_FAT32, 0, work, sizeof(work));
```

Comment:
This code is common to all ARM platforms, perhaps it's possible to add a function and avoid redundant code?